PWGCF/TwoParticleCorrelations/Tasks/longrangeCorrelation.cxx

```diff
@@ -180,10 +173,168 @@ struct LongrangeCorrelation {
+      histos.add("Fv0Global/ME/FV0Amp", "fv0mult", kTH2D, {channelFt0aAxis, amplitudeFt0a});
+      histos.add("Fv0Global/ME/FV0eta", "fv0;#eta", kTH1D, {axisEtaAssoc});
+      histos.add("Fv0Global/ME/FV0phi", "fv0;#phi", kTH1D, {axisPhi});
+      histos.add("Fv0Global/ME/FV0etavsphi", ";fv0;#eta;#phi", kTH2D, {axisPhi, axisEtaAssoc});
+      histos.add("Fv0Global/ME/deltaEta_deltaPhi", ";#delta#eta;#delta#phi", kTH2D, {axisDeltaPhi, axisDeltaEta});
+
+      sameFv0Global.setObject(new CorrelationContainer(Form("sameEventFv0Global_%i_%i", static_cast<int>(cfgMinMult), static_cast<int>(cfgMaxMult)), Form("sameEventFv0Global_%i_%i", static_cast<int>(cfgMinMult), static_cast<int>(cfgMaxMult)), corrAxis, effAxis, userAxis));
+      mixedFv0Global.setObject(new CorrelationContainer(Form("mixedEventFv0Global_%i_%i", static_cast<int>(cfgMinMult), static_cast<int>(cfgMaxMult)), Form("mixedEventFv0Global_%i_%i", static_cast<int>(cfgMinMult), static_cast<int>(cfgMaxMult)), corrAxis, effAxis, userAxis));
+    }
+
+    if (doprocessFv0MftSE || doprocessFv0MftME) {
+      histos.add("Fv0Mft/SE/hMult", "event multiplicity", kTH1D, {axisMultiplicity});
+      histos.add("Fv0Mft/SE/Trig_etavsphi", ";#eta;#phi", kTH2D, {axisPhi, axisEtaTrig});
+      histos.add("Fv0Mft/SE/Trig_eta", "#eta", kTH1D, {axisEtaTrig});
+      histos.add("Fv0Mft/SE/Trig_phi", "#eta", kTH1D, {axisPhi});
+      histos.add("Fv0Mft/SE/Trig_pt", "p_{T}", kTH1D, {axisPtTrigger});
+      histos.add("Fv0Mft/SE/hMult_used", "event multiplicity", kTH1F, {axisMultiplicity});
+      histos.add("Fv0Mft/SE/Trig_hist", "trig hist", kTHnSparseF, {axisSample, axisVtxZ, axisPtTrigger});
+      histos.add("Fv0Mft/SE/FV0Amp", "fv0mult", kTH2D, {channelFt0aAxis, amplitudeFt0a});
+      histos.add("Fv0Mft/SE/FV0eta", "fv0;#eta", kTH1D, {axisEtaAssoc});
+      histos.add("Fv0Mft/SE/FV0phi", "fv0;#phi", kTH1D, {axisPhi});
+      histos.add("Fv0Mft/SE/FV0etavsphi", ";fv0;#eta;#phi", kTH2D, {axisPhi, axisEtaAssoc});
+      histos.add("Fv0Mft/SE/deltaEta_deltaPhi", ";#delta#eta;#delta#phi", kTH2D, {axisDeltaPhi, axisDeltaEta});
+
+      histos.add("Fv0Mft/ME/hMult", "event multiplicity", kTH1D, {axisMultiplicity});
+      histos.add("Fv0Mft/ME/Trig_etavsphi", ";#eta;#phi", kTH2D, {axisPhi, axisEtaTrig});
+      histos.add("Fv0Mft/ME/Trig_eta", "#eta", kTH1D, {axisEtaTrig});
+      histos.add("Fv0Mft/ME/Trig_phi", "#eta", kTH1D, {axisPhi});
+      histos.add("Fv0Mft/ME/Trig_pt", "p_{T}", kTH1D, {axisPtTrigger});
+      histos.add("Fv0Mft/ME/FV0Amp", "fv0mult", kTH2D, {channelFt0aAxis, amplitudeFt0a});
+      histos.add("Fv0Mft/ME/FV0eta", "fv0;#eta", kTH1D, {axisEtaAssoc});
+      histos.add("Fv0Mft/ME/FV0phi", "fv0;#phi", kTH1D, {axisPhi});
+      histos.add("Fv0Mft/ME/FV0etavsphi", ";fv0;#eta;#phi", kTH2D, {axisPhi, axisEtaAssoc});
+      histos.add("Fv0Mft/ME/deltaEta_deltaPhi", ";#delta#eta;#delta#phi", kTH2D, {axisDeltaPhi, axisDeltaEta});
+
```

Comment:
Replicating code is error prone, difficult to maintain and hard to evolve
I would suggest unifying this five times replicated section of code
For inspiration you can check the `RealisticExample` task in https://github.com/AliceO2Group/O2Physics/blob/master/Tutorials/src/histogramRegistry.cxx